docs/SECURITY.md

```diff
@@ -4,7 +4,7 @@
 
 If there are any vulnerabilities in this project, don't hesitate to _report them_.
 
-1. Use any of the [private contact addresses](https://github.com/srealmoreno/rae-tesis/#Soporte).
+1. Use any of the [private contact addresses](mailto:security-report@raetesis.ml).
```

Comment:
Good job 👍🏿, but don't you think that a more descriptive email as **report-vulnerability@raetesis.ml** seems like a better idea?

Reply:
You're right, I'll do it.